packages/askui-nodejs/src/lib/ui-controller-facade.ts

```diff
@@ -113,10 +119,14 @@ export abstract class UiControllerFacade {
     return binarysizeInMB > sizeThresholdInMB;
   }
 
-  private async getBinary(binaryVersion: string, overWriteBinary = false): Promise<void> {
+  private async getBinary(
+    binaryVersion: string,
+    overWriteBinary = false,
+    proxyAgent?: { http: http.Agent, https: https.Agent },
```

Comment:
Same here, the agents themselves seem optional as well, and why not provide a default then

Reply:
obsolete, see https://github.com/askui/askui/pull/201#discussion_r1025314091